assets/js/home_controller.js

```diff
@@ -28,7 +28,9 @@ angular.module('ionicApp')
         $scope.queryVerse={};//trick for scope - detailed search compatibiliry. object needed
         $scope.queryVerse.keyword="";
         $scope.detailedSearchAuthorSelection = [];
-
+        $scope.peoplesearch = "";
+        $scope.circlesname = [];
```

Comment:
Bu sadece arama yapmak için ve sadece mobil arayüzde yardımcı olmak için olan bir değişken.
mobileCircleListForSelection desek iyi olur